src/components/ContactCard/ContactCard.spec.jsx

```diff
@@ -8,7 +8,7 @@ import {
 } from '../../helpers/doctypes'
 
 describe('ContactCard', () => {
-  it('should match snapshot', () => {
+  it('should match snapshot without groups', () => {
```

Comment:
Why `without groups`? There are groups in this snapshot..?

Reply:
Oh you're right, it should probably something like: 

it should match snapshot when groups are available but no one is linked to the specified contact